f5router/f5router.go

```diff
@@ -267,6 +305,23 @@ func (r *F5Router) Run(signals <-chan os.Signal, ready chan<- struct{}) error {
 		r.processCachedDataGroup(dg)
 	}
 
+	// See if there is an existing data group on the BIG-IP that is used to store
+	// bind ID -> route URI : plan ID information so the broker does not end up in
+	// a bad state on restart after a crash
+	if r.c.BrokerMode {
+		dg, err = fetchExistingDataGroup(r.c, BrokerDataGroupName)
```

Comment:
fetchExistingDataGroup should have retry back off loop for temporary and timeout net errors